### PR TITLE
Ensure `make test` downloads test packages

### PR DESCRIPTION
**Pull request by christophermanning**
Add `-t` flag to `go get` in `make test` so test packages are downloaded

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # Change Log
 
-## Unreleased
-
-### Changes
```
Comment (jeffreyc) [inline CHANGELOG.md:5]: Per http://keepachangelog.com/ (referenced in our other OSS projects' change logs), we should keep the subsections (probably "Added" and "Fixed" here).

**Comment by jeffreyc** [comment]
With all this new test coverage, this is a great time to add go-makefile to Travis.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: ruby
+
+services:
+  - docker
+
+install: git clone --depth 1 https://github.com/sstephenson/bats.git
```
Comment (jeffreyc) [inline .travis.yml:8]: Should `go-makefile` submodule `bats`? Or is the assumption that most people interested in running tests will already have `bats` installed?

File: GoMakefile [hunk 3]
```diff
@@ -12,6 +12,8 @@ ifndef DOCKER_IMAGE
 endif
 DOCKER_RUN := docker run --rm -v $(shell pwd):$(SRC) -w $(SRC) -e GITHUB_TOKEN $(DOCKER_IMAGE)
 
+REPRODUCABLE_TAR := --mtime='1970-01-01 00:00Z' --owner=root --group=root --numeric-owner
```
Comment (jeffreyc) [inline GoMakefile]: Reproducible (x3).

File: .travis.yml (hunk 2 above)
Comment (christophermanning) [inline .travis.yml:8]: I don't think it's appropriate to submodule bats into this project. People interested in running tests will have to install bats.

**Comment by jeffreyc** [comment]
LGTM.

**Pull request merged**